Content.Server/Atmos/EntitySystems/AtmosphereSystem.Processing.cs

```diff
@@ -467,6 +467,66 @@ private bool ProcessSuperconductivity(GridAtmosphereComponent atmosphere)
             return true;
         }
 
+        /// <summary>
+        /// Processes all entities with a <see cref="DeltaPressureComponent"/>, doing damage to them
+        /// depending on certain pressure differential conditions.
+        /// </summary>
+        /// <returns>True if we've finished processing all entities that required processing this run,
+        /// otherwise, false.</returns>
+        private bool ProcessDeltaPressure(Entity<GridAtmosphereComponent, GasTileOverlayComponent, MapGridComponent, TransformComponent> ent)
+        {
+            var atmosphere = ent.Comp1;
+            var count = atmosphere.DeltaPressureEntities.Count;
+            if (!atmosphere.ProcessingPaused)
+            {
+                atmosphere.DeltaPressureCursor = 0;
+                atmosphere.DeltaPressureDamageResults.Clear();
+            }
+
+            var remaining = count - atmosphere.DeltaPressureCursor;
+            var batchSize = Math.Max(50, DeltaPressureParallelProcessPerIteration);
```

Comment:
The fact that it clamps at 50 here is not documented anywhere. I can suggest removing this and relying entirely on the CVar.

Reply:
Also, the name of this variable is confusing because BatchSize is the name of a CVar that does something slightly different.